modules/flann/include/opencv2/flann.hpp

```diff
@@ -103,6 +103,56 @@ using ::cvflann::KL_Divergence;
 
 /** @brief The FLANN nearest neighbor index class. This class is templated with the type of elements for which
 the index is built.
+
+`Distance` functor specifies the metric to be used to calculate the distance between two points.
+There are several `Distance` functors that are readily available:
+
+__cv::flann::L2_Simple__ - Squared Euclidean distance functor. This is the simpler, unrolled version.
```

Comment:
Please use proper doxygen links instead. Try `#` prefix before symbol names.

Reply:
Since the symbols are defined in the `cvflann` namespace, they only turn to links if I prefix them with `cvflan::` as in `cvflann::L2_Simple`. I did not use these links because I wasn't sure if `cvflann` namespace should be exposed.

Reply:
Right. It seems that doxygen can't handle "using" statements.

This should work:
```
@link cvflann::L2_Simple cv::flann::L2_Simple @endlink
```

P.S. Direct usage of `cvflann` namespace should be avoided in user code (it is just a copy of original flann).

Reply:
Thank you for the suggestion.